_assets/scripts/gen_config.sh

```diff
@@ -10,7 +10,7 @@ LISTEN_PORT="${LISTEN_PORT:-30303}"
 API_MODULES="${API_MODULES:-eth,web3,admin}"
 MAX_PEERS="${MAX_PEERS:-50}"
 DAYS_KEPT="${DAYS_KEPT-30}"
-FLEET_NAME="${FLEET_NAME:-eth.prod}"
+FLEET_NAME="${FLEET_NAME:-shards.test}"
```

Comment:
So we're essentially replacing waku v1 with waku v2?
Can this not wait until we rename `shards.test` to `status.prod`?

Reply:
It's not a big deal, we na just change it later tho. Not sure how long the renaming will drag.

Reply:
I think renaming is pretty much done on fleet side, now? @jakubgs @richard-ramos

Reply:
I have finished renaming `shards.test` into `status.prod`. We just need to confirm it works as expected.